src/middleware/backend-authentication.js

```diff
@@ -19,17 +23,19 @@ module.exports = appName => {
 			req.path.indexOf('/__') === 0) {
 			next();
 		} else if (
+			// try keys first, falling back to IP whitelist
 			backendKeys.indexOf(req.get('FT-Next-Backend-Key')) > -1 ||
-			backendKeys.indexOf(req.get('FT-Next-Backend-Key-Old')) > -1
+			backendKeys.indexOf(req.get('FT-Next-Backend-Key-Old')) > -1 ||
+			ipWhitelist.validate(req.connection.remoteAddress)
 		) {
 			res.set('FT-Backend-Authentication', true);
 			next();
 		} else {
 			res.set('FT-Backend-Authentication', false);
 			if (process.env.NODE_ENV === 'production') {
 				// NOTE - setting the status text is very important as it's used by the CDN
-				// to trigger stale-if-error if we mess up the key synchronisation again
-				res.status(401).send('Invalid Backend Key');
+				// to trigger stale-if-error
+				res.status(401).send('Invalid Backend Authentication');
```

Comment:
OK good. But I think that PR should accept either value for a while as you won't be able to roll out to all the apps + the CDN simultaneously